feature-toggle/src/app/drawer/drawer.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { FeatureService } from '../feature.service';
+import { CommonModule } from '@angular/common';
+
+
+@Component({
+  selector: 'app-drawer',
+  standalone: true,
+  imports: [CommonModule],
```

Comment:
avoid using CommonModule, use @if, @for etc

Reply:
> avoid using CommonModule, use @if, @for etc

noted from yesterday's discuss